### PR TITLE
ADEN-11764 Fix Nativo regression after implementing NoAdsExperimentSetup

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

**Comment by preludek** [comment]
With this implementation, nativo AdSlots are being created twice which causes some undesired side effects like for example sending 2 success events.

File: src/ad-engine/providers/nativo/nativo.ts [hunk 1]
```diff
@@ -114,6 +114,12 @@ export class Nativo {
 			communicationService.getGlobalAction(eventsRepository.AD_ENGINE_SLOT_EVENT)(payload),
 		);
 	}
+  
+  private isDisabledInNoAdsExperiment(slotName): boolean {
+		const experimentUnit = this.context.get('state.noAdsExperiment.unitName');
```
Comment (preludek) [inline src/ad-engine/providers/nativo/nativo.ts:119]: I can't find where this is set. It seems like it is missing and it breaks the current implementation.

**Review by preludek — CHANGES_REQUESTED** [review]

**Review by preludek — APPROVED** [review]

File: spec/platforms/shared/setup/noads-experiment.setup.spec.ts [hunk 2]
```diff
@@ -21,8 +22,9 @@ describe('NoAdsExperimentSetup', () => {
 	const configGetSpy = sandbox.spy(instantConfig, 'get');
 	const cookieGetSpy = sandbox.spy(cookieAdapter, 'getItem');
 
-	afterEach(function() {
+	afterEach(function () {
 		sandbox.restore();
+		context.remove('state.noAdsExperiment.unitName');
```
Comment (preludek) [inline spec/platforms/shared/setup/noads-experiment.setup.spec.ts:27]: <3

File: platforms/shared/setup/noads-experiment.setup.ts [hunk 3]
```diff
@@ -58,6 +58,8 @@ export class NoAdsExperimentSetup implements DiProcess {
 			return;
 		}
 
+		context.set('state.noAdsExperiment.unitName', unitName);
```
Comment (preludek) [inline platforms/shared/setup/noads-experiment.setup.ts:61]: As long as beacons don't overlap it's 👍  we just have to remember to adjust it if we want to use that mechanism for more than 1 slot at a time

**Review by preludek — APPROVED** [review]

File: platforms/shared/setup/noads-experiment.setup.ts (hunk 3 above)
Comment (nandy-andy) [reply platforms/shared/setup/noads-experiment.setup.ts:61]: Definitely, if we want to cross-check the ad units we'll need to make it an array ;)

**Pull request merged**